### DISCUSSION TITLE
MIES does not show data in the databrowser

### DISCUSSION THREAD

**Discussion by t-b** [AllenInstitute/MIES#792] [General]
From Pedro Espinosa:

> Sorry the late reply. In fact I cannot acquire, the data browser is empty, and after press the button acquire data the 
> NaN sweeps became to numbers but no data. I think is something with the sweep counting, because also I cannot see > the amount of sweeps remaining. I leave you some screen shots of the problem.

![grafik](https://user-images.githubusercontent.com/1185677/105044969-a74e7400-5a67-11eb-9d47-a8153c31dd49.png)

![grafik](https://user-images.githubusercontent.com/1185677/105044988-ac132800-5a67-11eb-9848-2abc326f5730.png)

**Comment by p-espinosa** [comment]
Here the file

https://drive.google.com/file/d/1MkCBdnIPpE-Cn6x18KGsYnZW8hfCjBNq/view?usp=drive_web

**Reply by t-b** [reply]
@p-espinosa You have acquired on two headstages with the Testpulse simset selected. This means you are running the testpulse during data acquisition. This type of acquisition data is not displayed in the databrowser.

For acquiring real data acquisition you need to create a stimulus set in the wave builder, select that in the DAEphys panel and then you can watch that in databrowser.

**Reply by p-espinosa** [reply]
Thanks. Yes actually that was the problem. Everything is working. Sorry for the late answer.

Just an issue with the the TTL and test pulse. After each sweep I have a test pulse and a TTL, this is the background acquisition? if so, how I can remove the trigger of the TTL because I should not deliver stimulation between the sweeps. 

Thanks, and I can open another discussion regarding this issue if it is not clear.

**Reply by t-b** [reply]
Glad that this is resolved.

> After each sweep I have a test pulse and a TTL, this is the background acquisition?

I'm not understanding the question. Between sweeps you have a variable interval (inter trial interval ITI) and optionally you can run the testpulse during ITI. This is enabled by default, to turn that off choose "Settings->Activate TP during ITI". TTL stimsets are part of sweep data acquisition.

**Reply by p-espinosa** [reply]
Thanks to you. Now I notice that after a sweep finalise and the data is acquired and visible in the data browser. I have a "phantom" testpulse and TTL signal. It is not visible in Igor but I see it in my oscilloscope. Is right after the sweep finish.

**Reply by t-b** [reply]
Can you share the oscilliscope data? If possible as real data but a screenshot would work as well.

**Reply by p-espinosa** [reply]
The ITI is set at 10, the sweep is 1.2s, and the TTL is 10ms.

-In the file 1, the channel 1 is the TTL, channel 2 is a cell.

-In the file 2, channel 1 is a dead cell (still respond), channel 2 another cell, both respond to opt stimulation, so you can see the double stimulation, and also double test pulse.

-Also I added an screenshot of the data browser.

Basically at the end of each sweep, when the data is acquired, I have another test pulse and TTL. Like a phantom sweep.

[Data_Osciloscope.zip](https://github.com/AllenInstitute/MIES/files/6030328/Data_Osciloscope.zip)

**Reply by t-b** [reply]
I'll look into that in the next days.

**Reply by timjarsky** [reply]
@p-espinosa Could you share a NWB file from the experiment? - I don't see on in Data_Osciloscope.zip

**Reply by p-espinosa** [reply]
Hello, here is the NWB file. https://drive.google.com/file/d/1YdPSU2JOCJerCIWCVOLakbo4QnjyMaZj/view?usp=sharing

**Reply by t-b** [reply]
General remark: Please attach files directly into the post instead of a link. And screenshots can also be just added to the post so that one can see them directly.

The artefact you are seeing in the oscilloscope looks like an electronics connection artefact. Can you try to record the outputted TTL signal in MIES via an AD channel and check if the artefacts are then present as well?

**Reply by p-espinosa** [reply]
How can I do this? Because occur after the acquisition.

I always see this artefact in all the TTL outputs. I see all lights of the digital I/O in the acquisition card, illuminating after each sweep.

Didn't try with the latest version yet.
